src/sentry/incidents/logic.py

```diff
@@ -98,6 +98,12 @@
 
 CRITICAL_TRIGGER_LABEL = "critical"
 WARNING_TRIGGER_LABEL = "warning"
+INVALID_TIME_WINDOW = (
+    "Invalid time window for dynamic alert (valid windows are 15, 30, and 60 minutes)"
+)
+INVALID_ALERT_THRESHOLD = "Dynamic alerts cannot have a nonzero alert threshold"
+DYNAMIC_TIME_WINDOWS = {15, 30, 60}
+DYNAMIC_TIME_WINDOWS_SECONDS = {15 * 60, 30 * 60, 60 * 60}
```

Comment:
Could we make this populate based on `DYNAMIC_TIME_WINDOWS` like `DYNAMIC_TIME_WINDOWS_SECONDS {window * 60 for window in DYNAMIC_TIME_WINDOWS}` so we're not repeating the numbers in multiple places?

Same for `INVALID_TIME_WINDOW`, might read slightly worse but like 
`INVALID_TIME_WINDOW = f"Invalid time window for dynamic alert. Valid windows are {', '.join(map(str, DYNAMIC_TIME_WINDOWS))} minutes"`